tg.js

```diff
@@ -830,8 +830,8 @@ social.market.gift Magregalo ng KR
 social.market.gift.msg Mag-iwan ng mensahe
 respawn.auto Auto Respawning sa {0}
 custom.host.ranked Hindi ma-host habang naglalaro sa 'Ranked'
-custom.host.wager Can't host match while in Wager.
-funds.insufficient Hindi sapat na KR
+custom.host.wager Hindi pwede mag-host habang mayroong wager
+funds.insufficient Hindi sapat ang KR
 funds.out Ubos ang KR
```

Comment:
```suggestion
funds.out Ubos na ang iyong KR.
```

"It appears you have run out of Krunkies." was the original, it included second-person perspective